---
alumnum: false
permalink: /team/mark
level: 0
published: true

othernames: Mark
surname: Dunning
role: Bioinformatics Core Director
summary: Mark joined The University of Sheffield in October 2017 to establish the Bioinformatics Core. He works on a variety of different projects, and has a particular interest in Cancer Bioinformatics. He leads the development of our <a href="/training/">training program</a> and is an accredited Carpentries instructor and FHEA.
---
<p style="text-align:center;">
  <img src="/assets/images/people/Mark.jpg" width="250">
</p>


Mark joined The University of Sheffield in October 2017 to establish the Bioinformatics Core

- [twitter](https://twitter.com/DrMarkDunning)
- [github](https://github.com/markdunning)
- [orcid](https://orcid.org/0000-0002-8853-9435)

## Experience


2017 -... Bioinformatics Core Director, The University of Sheffield

2015 – 2017: Bioinformatics Training Co-ordinator, Cancer Research Uk, Cambridge Institute

2009 – 2015: Bioinformatics Analyst / Senior Bioinformatics Analyst, Cancer Research Uk Cambridge Institute

2005 – 2009: PhD (Oncology) University of Cambridge

2004 – 2005: Msc (Data Analysis, Networks and Nonlinear Dynamics) University of York

1999 – 2004: Bsc (Mathematics and Computer Science) University of York

 
## Biography

I obtained my PhD in the Statistics and Computational Biology group of [Simon Tavare](https://www.compbio.group.cam.ac.uk/) at The University of Cambridge. As part of my [thesis](https://www.repository.cam.ac.uk/handle/1810/218542) I developed open-source software for the analysis of Illumina microarray data, which is available through the [Bioconductor project](http://www.bioconductor.org). I joined the [Bioinformatics Core](https://www.cruk.cam.ac.uk/core-facilities/bioinformatics-core) at [Cancer Research Uk Cambridge Institute](https://www.cruk.cam.ac.uk/) and played a key role in the analysis of gene expression profiles as part of the [METABRIC](https://www.ncbi.nlm.nih.gov/pubmed/22522925) project, which identified and described new subtypes of breast cancer. I also participated in the pilot phases of the International Cancer Genome Consortium (ICGC) project by developing computational pipelines to process the whole-genome sequencing data from Oesophageal cancer patients. During my time in the Bioinformatics Core I also developed a passion for teaching and commenced a role dedicated to organising and delivering [Bioinformatics training courses](https://github.com/bioinformatics-core-shared-training), with the aim of empowering wet-lab scientists to begin to explore data for themselves and foster more-productive collaborations with Bioinformaticians.

I have a strong commitment to reproducible research and making my research outputs available to other researchers, and indeed members of the public who may have funded the research in the first place. For instance, I recently developed and deployed a [Shiny](https://shiny.rstudio.com/) application that allows interested parties to query various [prostate cancer datasets](https://bioinformatics.cruk.cam.ac.uk/apps/camcAPP/). In keeping with my open access principles, the code underlying the application is available via [github](https://github.com/crukci-bioinformatics/camcAPP) and utilises data sets that can be downloaded from [Bioconductor](https://bioconductor.org/packages/release/BiocViews.html#___ProstateCancerData). I have also recently investigated technologies such as Galaxy and [Docker](https://www.docker.com) to ease the deployment of software and facilitate reproducible research.

I am a certified instructor for [Data and Software Carpentry](https://carpentries.org/)

## Publications

For my updated and full list of publications, please see my ORCid entry:-

[https://orcid.org/0000-0002-8853-9435](https://orcid.org/0000-0002-8853-9435)

Selected Publications

1. Soulihol, C.,...Dunning, M.J,...and Evans, P.C. JAG1-NOTCH4 mechanosensing drives atherosclerosis. Science Advances (2022). 10.1126/sciadv.abo7958

2. Kariotis, S.,....Dunning, M.J….and Wang. D. Biological heterogeneity in idiopathic pulmonary arterial hypertension identified through unsupervised transcriptomic profiling of whole blood. (2021). Nature Communications. 10.1038/s41467-021-27326-0

3. Rhodes, C.J...., Dunning, M.J., ...Lawrie, A., Wilkins, M.R.  Whole Blood RNA Profiles Associated with Pulmonary Arterial Hypertension and Clinical Outcome. Am J Respir Crit Care Med. (2020) https://doi.org/10.1164/rccm.202003-0510OC

4. Yatai, K.B., Tran, H., Dunning, M.J., and Wang, D. Sources of Bias Within Consensus Subtypes of Muscle-invasive Bladder Cancer. European Urology 2019; doi:10.1016/j.eururo.2019.09.032.

5. Westbrook, J.A.,..Dunning, M.J.,,and Brown J.E., Identification and validation of DOCK4 as a potential biomarker for risk of bone metastasis development in patients with early breast cancer, The Journal of Pathology (2019),  https://doi.org/10.1002/path.5197

6. Dunning, M.J, Vowler, S.L., Lalonde, E., Ross-Adams. H,. Boutros, P., Mills, I.G., Lynch, A.G, and Lamb A.D., “Mining Human Prostate Cancer Datasets: The camcAPP Shiny App“ EBio Medicine 17, (2017)

7. Bakht, M.K,..Dunning, M.,..Porter, L.A., Neuroendocrine differentiation of prostate cancer leads to PSMA suppression, Endocrine-related cancer (2019) https://doi.org/10.1530/ERC-18-0226

8. Jurmeister, S.,..Dunning, M.J.,...and Neal, D.E. Identification of potential therapeutic targets in prostate cancer through a cross‐species approach, EMBO Molecular Medicine (2018) https://doi.org/10.15252/emmm.201708274


9. Ross-Adams, H., Lamb, A.D., Dunning, M.J., Halim, S., Lindberg, J., Massie, C.M., Egevad, L.A., Russell, R., Ramos-Montoya, A., Vowler, S.L., Sharma, N.L., Kay, J., Whitaker, H., Clark, J., Hurst, R., Gnanapragasam, V.J, Shah, N.C, Warren, A.Y., Cooper, C.S., Lynch, A.G., Stark, R., Mills, I.G., Gronberg, H. and Neal, D.E., “Integration of copy number and transcriptomics provides risk stratification in prostate cancer: A discovery and validation cohort study“ EBio Medicine 2(9):1133-1144, (2015).

10. Weaver, J.M.J., Shannon, N., Lynch, A.G., Ross-Innes, C.S., Forshew, T., Barbera, M., Ong, C.J., Lao-Sirieix, P., Dunning, M.J., Smith, L., Smith, M., Carvalho, B., O’Donovan. M., Underwood, T., Murtaza, M., May, A.P., Grehan, N., Hardwick, R., Davies, J., Olomi, A., Aparicio, S., Rozenfeld, N., Eldridge, M., Caldas, C., Edwards, P.A.W., Tavare, S. and Fitzgerald, R.C. “Ordering of mutations in preinvasive disease stages of esophageal carcinogenesis” Nature Genetics epub ahead of print (2014) ?

11. Dawson, S.J., Tsui, D.W.Y, Murtaza, M., Biggs, H., Rueda, O.M., Chin, S.F., Dunning, M.J., Gale, D., Forshew, T., Mahler-Araujo, B., Rajan, S., Humphray, S., Becq, J., Halsall, D., Wallis, M., Bentley, D., Caldas, C. and Rosenfeld, N., “Analysis of Circulating Tumour DNA to Monitor Metastatic Breast Cancer” The New England Journal of Medicine 368:1199-1209, 2013 ?

12. Curtis, C., Shah S., Chin, S.F., Turashvili, G., Rueda, O.M., Dunning, M.J., Speed, D., Lynch, A.G., Samarajiwa, S., Y, Yuan., METABRIC Group et al “The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.” Nature, 481(7381):389-93,2012 ?

13. Lynch, A.G., Chin, S.F., Dunning, M.J., Caldas, C., Tavare, S., and Curtis, C., “Calling sample mix-ups in cancer population studies” PLoS One, 7(8) e41815, 2012 ?

14. Ritchie, M.E., Dunning, M.J., Smith, M.L., Shi, W, and Lynch, A.G., “BeadArray expression analysis using bioconductor” PLoS Computational Biology 7(12) e1002276, 2011 ?

15. Dunning, M.J., Curtis, C., Barbosa-Morais, N.L., Caldas, C., Tavare, S. and Lynch, A.G., “The importance platform annotation in interpreting microarray data” Lancet Oncology 11(8):717, 2010 ?

16. Barbosa-Morais, N.L, Dunning, M.J., Samarajiwa, S.A., Darot, J.F.J., Ritchie, M.E., Lynch, A.G. and TavarE, S. “A re-annotation pipeline for Illumina BeadArrays: improving the interpretation of gene expression data” Nucleic Acids Research 38(3):e17, 2009 ?

17. Dunning, M.J., Barbosa-Morais, N.L, Lynch,. A.G., Tavare, S. and Ritchie, M.E. “Statistical issues in the analysis of Illumina data” BMC Bioinformatics 9:85, 2008 ?

18. Dunning, M.J. Smith, M.L., Ritchie, M.E. and Tavare, S. “beadarray: R classes and methods for Illumina bead-based data” Bioinformatics 23(16):2183-4 
