---
layout: page
organizer: Sheffield Bioinformatics Core
category: workshop
permalink: /training/rna-seq-introduction-2024-03-20
title: Introduction to RNA-Seq Analysis
excerpt: "Introduction to RNA-seq analysis from count data"
startDate: 2024-03-20
date: 2024-03-20
endDate: 2024-03-20
end-date: 2024-03-20
startTime: 13.00
from: 13:00
endTime: 16.00
to: 16:00
city: Sheffield, or online
postcode: S3 7ND
country: United Kingdom
location: "Bartolome House, Seminar Room EG03"
speaker: "Dr. Mark Dunning, Dr. Akshay Bhat"
venue: Bartolome House Seminar Room EG03, or ONLINE
description: "High-throughput RNA-sequencing is now the standard technique for quantifying transcript abundance in a biological sample of interest. In this course we will describe the processes that take place once you submit a library for RNA sequencing. We will describe the steps involved to go from sequencing counts to a list of genes that show statistically significant differences between your biological conditions of interest. Practical sessions will use the user-friendly Degust interface (https://degust.erc.monash.edu/) to demonstrate tasks such as quality control and assessing differential expression. We will also showcase some web sites you can use for enrichment and pathways analysis"
keywords: bioinformatics, NGS, RNA-seq, online, transcriptomics, functional genomics
scientific_keywords: bioinformatics, transcriptomics, functional genomics, RNA-seq
difficulty: beginner
contact: bioinformatics-core@sheffield.ac.uk
---

## Dates

Wednesday March 20th 2024 13:00 - 16:00

Online, and Bartolome House Seminar Room EG03, - University of Sheffield

## Registration

Please use [this link](https://onlineshop.shef.ac.uk/conferences-and-events/faculty-of-medicine-dentistry-and-health/medical-school) to register for the course. There are options for attending online and in-person. If you are based in Sheffield, we would strongly recommend attending in-person if possible.

## Course Materials

## Overview

The workshop will be *hands-on*; working through exercises and watching demonstrations from the lecturers.

High-throughput RNA-sequencing is now the standard technique for quantifying transcript abundance in a biological sample of interest. We will describe the steps involved to go from a count matrix to a list of genes that show statistically significant differences between your biological conditions of interest. Practical sessions will use the user-friendly Degust interface (https://degust.erc.monash.edu/) to demonstrate tasks such as quality control and assessing differential expression. We will also showcase some web sites you can use for enrichment and pathways analysis

*Please note that the course will not cover the analysis of RNA-seq data using the R programming langugage. We will also not explicitly cover the alignment of RNA-seq sequences against a reference genome*

## Who should attend this course?

Researchers in life sciences who want to get an appreciation for the computational steps involved in RNA-seq analysis, but not neccesarily wishing to execute the pipeline for themselves. 

## Objectives:- After this course you should be able to:

- Appreciate some of the issues that can arise when designing an RNA-seq experiment 
- Interpret an Multi-dimensional scaling (MDS) plot to check for batch effects and confounding factors
- Use Degust to generate a statistically robust gene list
- Perform a gene set enrichment analysis using available online tools

## Aims:- During this course you will learn about:

- Basic principles of Experimental design for RNA-seq experiments
- The steps in a best-practice pipeline for RNA-seq analysis
- Configuring the Degust interface to perform different types of differential expression
- The theory behind popular methods for pathways and gene set enrichment analysis


## Instructors

- [Mark Dunning](http://sbc.shef.ac.uk/team/mark/index.html), Bioinformatics Core Director
- [Emily Chambers](http://sbc.shef.ac.uk/team/emily/index.html), Bioinformatics Core Analyst


