---
layout: page
organizer: Sheffield Bioinformatics Core
title: Intro to Next Generation Sequencing and Bioinformatics
excerpt: "Intro to Next Generation Sequencing and Bioinformatics"
startDate: 2021-11-29
endDate: 2021-12-06
startTime: 13.00
endTime: 17.00
city: Sheffield
postcode: S3 7ND
country: United Kingdom
venue: ONLINE
description: "This course covers state-of-the-art and best-practice tools for the analysis of genomes and transcriptomes. We describe, and give hands-on experience of, the entire analysis workflow from raw data generated by a sequencing machine to deriving variant calls (e.g. Single Nucleotide Variants) that are ready for downstream analysis, interpretation and prioritisation. We will describe the steps involved to go from sequencing library to a prioritised, clinically-relevant list of DNA variants. Practical sessions will use the user-friendly Galaxy interface (https://usegalaxy.org/) to demonstrate tasks such as alignment, quality control, variant-calling and annotation."
keywords: bioinformatics, NGS, DNA-seq, variant-calling, RNA-seq
difficulty: beginner
contact: bioinformatics-core@sheffield.ac.uk
---



## Registration 

Please book your place on the course using [this link](https://onlineshop.shef.ac.uk/conferences-and-events/faculty-of-medicine-dentistry-and-health/neuroscience/introduction-to-next-generation-sequencing-and-bioinformatics)

# Overview

This course covers state-of-the-art and best-practice tools for the analysis of genomes and transcriptomes. We describe, and give hands-on experience of, the entire analysis workflow from raw data generated by a sequencing machine to deriving variant calls (e.g. Single Nucleotide Variants) or differentially-expressed genes that are ready for downstream analysis, interpretation and prioritisation. Practical sessions will use the user-friendly Galaxy interface (https://usegalaxy.org/) to demonstrate tasks such as alignment, quality control, differential expression, variant-calling and annotation **without requiring any previous Bioinformatics or coding experience**.

You can choose to attend either session 2 or 3 (or both) according to your researcher goals and interest. However, **attendance at session 1 is compulsory** as it will introduce many fundamental tools and concepts that are common to both sessions 2 and 3.

## Session 1 (**Compulsory**) - Introduction to NGS and Galaxy
 
Learning Objectives

- Import of data into Galaxy and Quality Control
- Understand the contents of *fastq* and *bam* files
- Quality assessment of sequencing data
- Alignment of sequences against a reference genome
- Visualisation of sequencing data in the Integrative Genomics Viewer (IGV)
 

## Session 2 (**Optional**) - Variant Calling

Learning Objectives

- Processing of aligned reads prior to variant-calling
- Identifying Single-nucleotide variants from aligned reads
- Visualising variant calls in IGV
- Annotating variants calls with respect to genomic coordinates
- Analysis workflows for matched tumour and normal samples

## Session 3 (**Optional**) - RNA-seq

Learning Objectives

- Producing a count matrix from a set of aligned reads
- Performing quality assessment and detecting outliers and batch effects
- Differential Expression using a web-tool ("Degust")
- Pathways and enrichment analysis

## Who should attend this course?

Researchers who are planning NGS experiments in the future and want to gain an appreciation for how the workflows and analyses are executed, and what the expected outputs are. 


## Prerequisites

- No prior programming experience is required. Prior knowledge of NGS sequencing technologies would be an advantage, but not required.

## Instructors


- Dr Emily Chambers, Bioinformatics Core Scientist
- Dr Mark Dunning, Bioinformatics Core Director




