---
layout: page
organizer: Sheffield Bioinformatics Core
title: RNA-seq Analysis for Beginners
excerpt: "Intro to RNA-seq"
description: "High-throughput RNA-sequencing is now the standard technique for quantifying transcript abundance in a biological sample of interest. In this course we will describe the processes that take place once you submit a library for RNA sequencing, and what data you should expect to receive from the Bioinformatics Core. We will describe the steps involved to go from sequencing library to a list of genes that show statistically significant differences between your biological conditions of interest. Practical sessions will use the user-friendly Galaxy interface (https://usegalaxy.org/) to demonstrate tasks such as alignment, quality control and assessing differential expression. We will also showcase some web sites you can use for enrichment and pathways analysis."
keywords: bioinformatics, NGS, RNA-seq, galaxy
difficulty: beginner
contact: bioinformatics-core@sheffield.ac.uk
---

## Overview

High-throughput RNA-sequencing is now the standard technique for quantifying transcript abundance in a biological sample of interest. In this course we will describe the processes that take place once you submit a library for RNA sequencing, and what data you should expect to receive from the Bioinformatics Core.

We will describe the steps involved to go from sequencing library to a list of genes that show statistically significant differences between your biological conditions of interest. Practical sessions will use the user-friendly Galaxy interface (https://usegalaxy.org/) to demonstrate tasks such as *alignment*, *quality control* and *assessing differential expression*. We will also showcase some web sites you can use for enrichment and pathways analysis.

*Please note that the course will not cover the analysis of RNA-seq data using the  R programming langugage*



## Setup

- Please register for an account at [usegalaxy.eu](https://usegalaxy.eu).**Make sure to check your email to activate your account**
- Please download the [Integrative Genomics Viewer](https://software.broadinstitute.org/software/igv/download)
- Please download the javaGSEA file from 
[http://software.broadinstitute.org/gsea/downloads.jsp](http://software.broadinstitute.org/gsea/downloads.jsp) (gsea3.0-jar)
- Please watch these pre-recorded lectures
  + [Introduction and Experimental Design](https://youtu.be/YtOG-77FfIg)
  + [RNA-seq workflow overview](https://youtu.be/m1x4CaLg90A)
  + [Functional Gene Analysis](https://youtu.be/clb0bh3zFSM)


## Course Notes

+ [RNA-seq Pre-processing](http://sbc.shef.ac.uk/rnaseq-introduction-online/01-pre-processing.nb.html)
+ [Differential Expression](http://sbc.shef.ac.uk/rnaseq-introduction-online/02-differential-expression.nb.html)
+ [Gene Set Enrichment](http://sbc.shef.ac.uk/rnaseq-introduction-online/03-enrichment.nb.html)
  
## Video Recordings

Videos of the instructor going through the course materials will be available on youtube



## Who should attend this course?

Researchers in life sciences who want to get an appreciation for the computational steps involved in RNA-seq analysis, but not neccesarily wishing to execute the pipeline for themselves. 

## Objectives:- After this course you should be able to:

- Appreciate some of the issues that can arise when designing an RNA-seq experiment 
- Recall the key pipeline steps in an RNA-seq analysis
- Understand the contents of fastq and bam file format for RNA-seq data
- Perform a gene set enrichment analysis using available online tools

## Aims:- During this course you will learn about:

- Basic principles of Experimental design for RNA-seq experiments
- The steps in a best-practice pipeline for RNA-seq analysis
- The theory behind popular methods for pathways and gene set enrichment analysis
- Executing basic Bioinformatics tools using Galaxy

## Software installation

Please download the Integrative Genomics Viewer (IGV) for your operating system using [this link](https://software.broadinstitute.org/software/igv/download) and also download the javaGSEA file from 
[http://software.broadinstitute.org/gsea/downloads.jsp](http://software.broadinstitute.org/gsea/downloads.jsp) (gsea3.0-jar)
