---
title: What expertise can SBC offer?
permalink: /collaboration/expertise
slug: index
type: text
carousels:
  - images: 
    - image: /assets/images/nf-core-rnaseq_metro_map_grey.png
    - image: /assets/images/example_volcano.png    
    - image: /assets/images/example_heatmap.png
    - image: /assets/images/example_genetonic1.png
    - image: /assets/images/example_genetonic2.png
    - image: /assets/images/example_umap.png
    - image: /assets/images/spt.png
    - image: /assets/images/example_rainfall.png
    - image: /assets/images/example_oncoplot.png
    - image: /assets/images/example_go.png
    - image: /assets/images/example_network_analysis.png
    - image: /assets/images/hsa04110.pathview.png
---
{% include carousel.html height="50" unit="%" duration="5" number="1" %}

The SBC has expertise in the following areas, and software packages. If you are interested in some software or methodologies not listed here, please ask.

## Experimental Design
If you are planning an experiment, please come to us before generating data so that we can discuss the design...
<details>
  <summary>See more</summary>

The Core is experienced in advising on the following issues:-
<ul class="b">
<li>Optimal number of replicates</li>
<li>Biases and confounding factors</li>
<li>Single-end or paired-end sequencing</li>
<li>Depth of sequencing required</li>
<li>Contacts at Sequencing vendors</li>

For experimental advice on performing high-throughput studies, please contact the <a href="https://www.sheffield.ac.uk/sitran/facilities/multiomics">SITraN multi-omics facility</a> or <a href="https://sites.google.com/sheffield.ac.uk/genomics-laboratory/home">Genomics Facility in Biosciences</a>.
</ul>
</details>


## Next Generation Sequencing (NGS) Analysis

<details>
  <summary>See more...</summary>
  <ul class="b">
<li> Retrieval and analysis of public datasets on Gene Expression Omnibus (GEO) and Sequencing Read Archive (SRA) </li>
<li> BWA, and other aligners for all NGS platforms and applications </li>
<li> edgeR, DESeq2, STAR and salmon (among others) for RNA-Seq analysis and differential expression</li>
<li> Clustering and PCA to identify batch effects </li>
<li> Downstream analysis of gene lists to identify over-represented pathways / ontologies using GSEA, goseq, or other online tools such as DAVID</li>
<li> Alternative splicing analysis and visualization (e.g. sashimi plots)</li>
<li> GATK, VarScan and MuTect for somatic and germline mutation calling </li>
<li> MACS2, and other peak callers for ChIP-Seq and ATAC-seq analysis </li>
<li> Submission of data to public repositories for publication </li>

</ul>
</details>


## Single Cell RNA-Seq

<details>
  <summary>See more...</summary>
  <ul class="b">
<li> Running cellranger</li>
<li> QC of single-cell RNA-seq and filtering poor quality cells / samples</li>
<li> Dimensionality reduction and plotting with UMAP, TSNE etc</li>
<li> Marker gene identification</li>
<li> Analysis of Spatial transcriptomics data</li>
</ul>
</details>

## Microarray Analysis
<details>
  <summary>See more...</summary>
<ul class="b">
<li>Experience with data from gene expression data from Affymetrix and Illumina platforms</li>
<li> Re-analysing public data deposited on GEO or ArrayExpress </li>
<li>Experience with varying data types, including expression, methylation (450K), SNP and copy-number</li>
</ul>
</details>

## Further Expertise

<details>
  <summary>See more...</summary>

<ul class="b">
<li>Custom visualisation of high-throughput data</li>
<li>Implementing and running workflows and piplines with <a href="https://www.nextflow.io/">nextflow</a></li>
<li>Packaging and deploying software environments with <a href="https://www.docker.com/">docker</a></li>
<li> Data Management and curation</li>
<li> Creating interactive web applications wih <a href="https://shiny.rstudio.com/">Shiny</a></li>
</ul>
</details>

